nullaway/src/main/java/com/uber/nullaway/jarinfer/JarInferStubxProvider.java

```diff
@@ -0,0 +1,19 @@
+package com.uber.nullaway.jarinfer;
+
+import java.util.List;
+
+/** Interface to be implemented by providers of stubx files to be read by NullAway */
+public interface JarInferStubxProvider {
```

Comment:
@lazaroclapp one annoying thing is any jar wanting to provide stubx files needs to add at least a `compileOnly` dependence on the main NullAway artifact.  I'm wondering, should we move the `LibraryModels` interface and this one to a new small "providers" artifact, so that others could just depend on that?

Reply:
Ok, cool, yes, we can do this in a follow-up.